package.json

```diff
@@ -11,7 +11,18 @@
   "configSchema": {
     "executablePath": {
       "type": "string",
-      "default": "codeclimate"
+      "default": "/usr/local/bin/codeclimate",
```

Comment:
And restoring the default being part of that is why I put the comment on here, so it would be hidden when this gets fixed 😉.